src/client/invite/Invite.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import { Form, Input } from 'antd';
+import Affix from '../components/Utils/Affix';
+import LeftSidebar from '../app/Sidebar/LeftSidebar';
+import requiresLogin from '../auth/requiresLogin';
+import { getAuthenticatedUserName } from '../reducers';
+import './Invite.less';
+
+@requiresLogin
+@connect(
+  state => ({
+    authenticatedUserName: getAuthenticatedUserName(state),
+  }),
+)
+export default class Invite extends React.Component {
+  static propTypes = {
+    authenticatedUserName: PropTypes.string,
+  };
+
+  static defaultProps = {
+    authenticatedUserName: '',
+  };
+
+  render() {
+    const { authenticatedUserName } = this.props;
+    let inviteURL;
+    if (typeof window !== 'undefined') {
```

Comment:
If this page gets rendered on server `inviteURL` will be null and will stay this way (so there will be no invite URL at all).
`compontentDidMount` is not called on the server, only after mounting in a web browser. We could use this to create URL because `window` should be initialised there. We will save it in state (eslint is not going to like this, but we can create new method `createInviteURL` so eslint won't warn about using setState in componentDidMount.